zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -487,21 +492,22 @@ mod tests {
                          END
                        ) AS memo_count
                 FROM sent_notes
+                JOIN transactions
+                     ON transactions.id_tx = sent_notes.tx
                 LEFT JOIN sapling_received_notes
                           ON (sent_notes.tx, sent_notes.output_pool, sent_notes.output_index) =
                              (sapling_received_notes.tx, 2, sapling_received_notes.output_index)
                 WHERE  sapling_received_notes.is_change IS NULL
                    OR  sapling_received_notes.is_change = 0
-                GROUP BY account_id, id_tx
+                GROUP BY account_id, txid
             ),
             blocks_max_height AS (
                 SELECT MAX(blocks.height) as max_height FROM blocks
             )
             SELECT notes.account_id                  AS account_id,
-                   transactions.id_tx                AS id_tx,
-                   transactions.block                AS mined_height,
+                   notes.block                       AS mined_height,
+                   notes.txid                        AS txid,
```

Comment:
```suggestion
                   notes.txid                        AS txid,
                   notes.block                       AS mined_height,
```
So that we more closely follow the release notes' "The `v_transactions` and `v_tx_outputs` views now return the 32-byte transaction identifier for transactions instead of the internal database id." (The ordering is visible.)